snap/info.go

```diff
@@ -649,17 +649,17 @@ func DataHomeGlobs(opts *dirs.SnapDirOptions) []string {
 		opts = &dirs.SnapDirOptions{}
 	}
 
-	list := make([]string, len(dirs.GetSnapHomeDirs()))
-	copy(list, dirs.GetSnapHomeDirs())
-	for i, dir := range list {
+	var homeDirsSlice []string
```

Comment:
tiny nitpick, I don't think we need to add `Slice` postfix, please feel free to ignore.